src/main/java/seedu/address/logic/commands/AddMemshipCommand.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MEMSHIP;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Membership;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a membership tier for a person to the address book.
+ */
+public class AddMemshipCommand extends Command {
```

Comment:
Would it be better if the command was changed from `AddMemship` to `ChangeMemship`? Also, might be better to name this class the full name `ChangeMembershipCommand`, but the `COMMAND_WORD` can stay as `changemship`!

Reply:
Okay good idea, will do in the next iteration